test/chrono-test.cc

```diff
@@ -6,7 +6,9 @@
 // For the license information refer to format.h.
 
 #ifdef WIN32
```

Comment:
Let's get rid of WIN32 for simplicity. There is no harm in having the macro always defined.